.github/pr-blocking-todos/checkfortodos.py

```diff
@@ -0,0 +1,64 @@
+import sys
+
+def checkForTodos(fnameToCheck):
```

Comment:
This code doesn't have any unit tests.  Not a tragedy, but it would probably be good practice to include them.